core/src/test/java/org/elasticsearch/search/aggregations/bucket/significant/SignificantLongTermsTests.java

```diff
@@ -35,22 +37,22 @@
 import java.util.Map;
 import java.util.Set;
 
-import static org.elasticsearch.search.aggregations.InternalAggregations.EMPTY;
-
 public class SignificantLongTermsTests extends InternalSignificantTermsTestCase {
 
     private SignificanceHeuristic significanceHeuristic;
+    private DocValueFormat format;
 
     @Before
     public void setUpSignificanceHeuristic() {
         significanceHeuristic = randomSignificanceHeuristic();
+        format = randomNumericDocValueFormat();
```

Comment:
ok weird that it worked. does the format have a default value in case this method is not called?